### PR TITLE
[Issue_tracker]  bug fix for 20.3-release | can't edit an issue

### PR DESCRIPTION
**Pull request by kongtiaowang**
### Brief summary of changes
Give an issueID on url.

### This resolves issue...
https://redmine.cbrain.mcgill.ca/issues/16241

### To test this change...
edit an issue

### CODE REVIEW

**Comment by kongtiaowang** [comment]
@ridz1208 
This PR is for the 20.3-release

File: modules/issue_tracker/jsx/columnFormatter.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ function formatColumn(column, cell, rowData, rowHeaders) {
   if (column === 'Title') {
     let cellLinks = [];
     cellLinks.push(
-      <a href={loris.BaseURL + '/issue_tracker/issue/' + row['Issue ID']}>
+      <a href={loris.BaseURL + '/issue_tracker/issue/?issueID=' + row['Issue ID']}>
```
Comment (driusan) [inline modules/issue_tracker/jsx/columnFormatter.js]: This is not correct. The ID should be coming from the handle method parsing the URL at https://github.com/aces/Loris/blob/minor/modules/issue_tracker/php/issue.class.inc#L144

Comment (kongtiaowang) [reply modules/issue_tracker/jsx/columnFormatter.js]: @driusan If use RESTful API in this case, the "QueryString.get()" will get a null value. 
Major branch is also using "?issueID=" on URL. I think if you need this to be fixed, it should on Major branch.

Comment (driusan) [reply modules/issue_tracker/jsx/columnFormatter.js]: in that case the major branch is wrong and also needs to be fixed. The ID should be coming from the URL, not the query string

Comment (ridz1208) [reply modules/issue_tracker/jsx/columnFormatter.js]: @driusan will look into the `handle` function and try to solve the issue

**Comment by ridz1208** [comment]
https://github.com/aces/Loris/pull/3942

File: modules/issue_tracker/jsx/columnFormatter.js (hunk 1 above)
Comment (kongtiaowang) [reply modules/issue_tracker/jsx/columnFormatter.js]: @driusan @ridz1208 could u review again?

**Pull request merged**